### DISCUSSION TITLE
Follow Cursor example available?

### DISCUSSION THREAD

**Discussion by J4v4Scr1pt** [floating-ui/floating-ui#1851] [React] [answered]
Hello!

First of truly awesome library this!

I am creating a tooltip and need to create the follow cursor feature avalible in popper.
From what I understand this commit add this functionallity: https://github.com/floating-ui/floating-ui/pull/1748

I know this commit was quite recently but because I am stupid is there some example how this is applied in practice anywhere? 😄 

Thanks in advance!

/Regards

**Comment by atomiks** [comment, marked as answer]
I need to make a React demo but this might help: https://codesandbox.io/s/fancy-worker-xkr8xl?file=/src/index.js

**Reply by J4v4Scr1pt** [reply]
That is perfect, thanks man!
Keep up the awesome work and have a superb day 😃!

**Reply by J4v4Scr1pt** [reply]
@atomiks sorry to bother you again, but I have some trouble with the tooltip with the cursor blocking the tooltip from sliding in from the bottom. I works on all the placements except the top.

Do you have any suggestions on this?

Here is a codebox showing the issue: https://codesandbox.io/s/currying-mountain-w6zezy?file=/src/Tooltip.jsx

**Reply by atomiks** [reply]
You'll want to put `pointer-events: none` on the tooltip as the tooltip is blocking pointer-events from firing on the reference once it appears

**Reply by J4v4Scr1pt** [reply]
Omg you mean that I build this and forget about something stupid as that.. I'm embarrassed.. So sorry for bothering you with this. 
Maybe someone atleast can get some help from the sandbox I created if they strugle with this..

Super grateful for your time! 
Take care!

/Regards
